technologies/job/r/r-3.4.4-base/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM rocker/tidyverse:3.4.4
+
+USER root
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV HADOOP_CONF_DIR=/etc/hadoop/conf
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/lib/hadoop/lib/native"
+
+# Install system libraries required by R packages
+RUN apt update -qq && apt install -qqy --no-install-recommends \
+      libcups2 libcups2-dev openjdk-8-jdk systemd \
+      unixodbc-dev libbz2-dev libgsl-dev odbcinst libx11-dev mesa-common-dev libglu1-mesa-dev \
+      gdal-bin proj-bin libgdal-dev libproj-dev libudunits2-dev libtcl8.6 libtk8.6 libgtk2.0-dev \
+      krb5-user \
+    && apt clean \
+    && rm -rf /var/lib/apt/lists/*;
+
+RUN mkdir /root/.R/ \
+    && echo CXXFLAGS=-DBOOST_PHOENIX_NO_VARIADIC_EXPRESSION > /root/.R/Makevars
+
+# Install Saagie's RStudio Addin
+RUN R -e "install.packages('devtools')" && \
+  R -e "devtools::install_github('saagie/rstudio-saagie-addin')"
+
+# Be sure rstudio user has full access to his home directory
+RUN mkdir -p /home/rstudio && \
+  chown -R rstudio:rstudio /home/rstudio && \
+  chmod -R 755 /home/rstudio
+
+# Install Impala ODBC dependency
```

Comment:
Pourquoi ne pas utiliser `IMPALA_ODBC_VERSION` comme dans l'image 3.6.2 ?

Reply:
this image like 3.5.3 remained mainly untouched for months whereas 3.6.2 has been kept up to date